task-list/app/controllers/tasks_controller.rb

```diff
@@ -1,16 +1,27 @@
 class TasksController < ApplicationController
 
+  def complete
+    @task = Task.find(params[:id])
+    @task.update(completed_date: Date.current)
+    redirect_to action: :index
```

Comment:
Awesome! I knew there was a way to tell `redirect_to` to go to another route by providing the action (and optionally the controller). Glad to see you were able to get it working.